Dockerfile.dev

```
@@ -2,7 +2,7 @@ FROM node:16
 WORKDIR /app
 
 # Setup pnpm package manager
-RUN npm install -g pnpm
+RUN npm install -g pnpm@7.6.0
```

I think Vercel uses 7.5.2 atm so we might want to stick to that for consistency?
Last week we had a problem with deployments because we accidentally updated pnpm to 7.6.0 locally but Vercel is lagging behind.

```suggestion
RUN npm install -g pnpm@7.5.2
```

---

Yes, it can help us see problems earlier (broken on Vercel -> we should check Docker too)